pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -919,6 +919,22 @@ var _ = Describe("Converter", func() {
 			})
 		})
 
+		DescribeTable("CPU mpx feature", func(arch string, hasMPX bool) {
```

Comment:
nit: instead of passing `hasMPX bool` you can pass `matcher types.GomegaMatcher` and just use it as `Expect(domain.Spec.CPU.Features).To(matcher)`, and passing in `HaveExactElements(api.CPUFeature{Name: "mpx", Policy: "disable"})` or `BeNil()` accordingly.